src/k8s/request.js

```diff
@@ -362,6 +362,33 @@ const getDevices = vm => {
   return domain.devices;
 };
 
+export const assignBootOrderIndex = (vm, currDevBootOrder = -1) => {
+  let bootOrder = currDevBootOrder;
+  if (currDevBootOrder !== BOOT_ORDER_FIRST) {
+    bootOrder = getNextAvailableBootOrderIndex(vm);
+  }
+  return bootOrder;
+};
+
+const getNextAvailableBootOrderIndex = vm => {
+  const disks = getDisks(vm);
+  const nics = getInterfaces(vm);
+  let largestIdx = -1;
+
+  disks.forEach(disk => {
+    const bootOrder = get(disk, 'bootOrder', -1);
+    largestIdx = bootOrder > largestIdx ? bootOrder : largestIdx;
+  });
+
+  nics.forEach(nic => {
```

Comment:
I would rather not set explicitly boot order for networks when creating VMs. Except the ones the user chooses. I know this concerns editing the VMs detail, so maybe we should have an option to discard some nics (or disks) from the boot order? Otherwise this behavior could have security ramifications.

Reply:
maybe there could be one component for boot order which could be used in VM details and in the create VM dialog? It could have its own step (something like BIOS?). So we would not be forced to set bootorder to all devices - just the ones user chooses. Thoughts?

Reply:
> maybe there could be one component for boot order which could be used in VM details and in the create VM dialog? It could have its own step (something like BIOS?). So we would not be forced to set bootorder to all devices - just the ones user chooses. Thoughts?

+1... I like this idea. @jelkosz @rawagner What do you think?

Reply:
sounds good to me!

Reply:
After a discussion, it was decided that everything would be bootable. The boot order will be the order of devices in CreateDialog. Afterwards, it can be edited in VM detail with possibility to change or remove the bootOrder.

Reply:
Boot order should be set when creating VMs. Created relevant issue: #230